web/src/main/java/ua/com/vertex/controllers/exceptionHandling/exceptions/MultipartValidationException.java

```diff
@@ -0,0 +1,7 @@
+package ua.com.vertex.controllers.exceptionHandling.exceptions;
+
+public class MultipartValidationException extends RuntimeException {
+    public MultipartValidationException(String message) {
```

Comment:
please add 'Throwable cause' parameter

Reply:
I don't exactly understand what you want here

Reply:
I mean adding one more method parameter with type 'Throwable' and name 'cause' - exception that would be mentioned as 'caused by' in stacktrace.

Reply:
Actually, I've took a look on it and it's fine for now. No additional parameters are needed